content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -53,7 +53,7 @@ You can configure a static Pod two different ways: either by using configuration
 
 ### Filesystem-hosted static Pod manifest {#configuration-files}
 
-The configuration files are just standard Pod definitions in JSON or YAML format in a specific directory. Use `kubelet --pod-manifest-path=<the directory>` to start the kubelet or add the `staticPodPath: <the directory>` field in the [KubeletConfiguration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
+The Manifests are just standard Pod definitions in JSON or YAML format in a specific directory. Use `staticPodPath: <the directory>` field in the [KubeletConfiguration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
```

Comment:
No cap for Manifests after The.
Suggestion: A manifest is a standard Pod definition ...
OR
Manifests are standard Pod definitions ...